src/compile/scale.ts

```diff
@@ -66,11 +97,12 @@ export function type(fieldDef: FieldDef, channel: Channel, mark: Mark): string {
     case NOMINAL:
       return 'ordinal';
     case ORDINAL:
+      if (channel === COLOR) {
+        return 'linear'; // time has order, so use interpolated ordinal color scale.
```

Comment:
(`scale.type` is currently being called by other files to satisfy some condition, can't remember what are those, but what this method returns might have implication for other parts of the codebase.